### PR TITLE
164796910 Adds Comment Edit History feature

### PR DESCRIPTION
**Pull request by micah-akpan**
#### Description
Add a comment edit history feature to facilitate the retrieval of a comment's edit history
#### Type of change
New feature (non-breaking change that adds functionality)
#### How Has This Been Tested?
- [x] unit tests
- [x] integration tests
#### Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have added tests that prove my fix is effective or that my feature works
#### Screenshots
- Postman:
(no edits yet)
comment edit history - empty
(new edits has been made)
comment-edit-history

- Flowchart
comment-history

#### PT
[164796910](https://www.pivotaltracker.com/story/show/164796910)

### CODE REVIEW

File: tests/unit/util.test.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ describe('Util test', () => {
       });
     });
 
-    it('should return error 2', () => {
+    it('should return error', () => {
```
[the remark this answers is not in the document]
Comment (micah-akpan) [reply tests/unit/util.test.js:86]: This has been fixed a long time ago. It should be a problem with HoundCI server

**Review by ssanusi — COMMENTED** [review]
LGTM

**Review by chikeozulumba — COMMENTED** [review]
Good implementation @micah-akpan 
LGTM

**Comment by Simplemart17** [comment]
LGTM

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**